### PR TITLE
PACT-713 Define schema for Edit Set Record

### PR DESCRIPTION
**Pull request by felixgb**
You can preview the schema using this tool: https://studio.asyncapi.com

### CODE REVIEW

File: schema/omega-services.async.api.yaml [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+    * Get an Edit Set 🪣
+    * Get a Record 📜
+    * Update a Record ✍️
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  contact:
+    name: Adam Retter
+    email: adam@evolvedbinary.com
+    url: https://www.evolvedbinary.com
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:25]: Can we change this to:-
```
name: The National Archives
email: webmaster@nationalarchives.gov.uk
url: https://www.nationalarchives.gov.uk/
```

**Review by rwalpole — CHANGES_REQUESTED** [review]

File: schema/omega-services.async.api.yaml [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        startDateMonth:
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        startDateYear:
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:273]: The start date will be an ISO 8601 date, as you have already supposed in the ticket. The frontend will need to handle the rendering of this date into the form fields and parsing from the same.

File: schema/omega-services.async.api.yaml [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        endDateMonth:
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        endDateYear:
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:288]: The end date will be an ISO 8601 date, as you have already supposed in the ticket. The frontend will need to handle the rendering of this date into the form fields and parsing from the same.

File: schema/omega-services.async.api.yaml [hunk 4]
```diff
@@ -0,0 +1,442 @@
...
+            people and places). Its purpose is to enable users to judge the
+            potential relevance of the record described and whether they want
+            to investigate the material further or not.
+
+            Note: Called "Scope and Content" in TNA-CS13
+        coveringDate:
+          type: string
+          description: |
+            Identifies and records the date(s) of creation of the records being
+            described.
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:252]: I agree with your comment in the ticket that this should be `coveringDates` as this is the term used in CS13 (catalogue standards document).

**Review by rwalpole — CHANGES_REQUESTED** [review]

File: schema/omega-services.async.api.yaml (hunk 2 above)
Comment (felixgb) [reply schema/omega-services.async.api.yaml:273]: OK, in the future after we change the frontend shall we come back to this and change it in the schema?

File: schema/omega-services.async.api.yaml (hunk 4 above)
Comment (felixgb) [reply schema/omega-services.async.api.yaml:252]: Changed to `coveringDates`, in both the record and the summary.

File: schema/omega-services.async.api.yaml (hunk 2 above)
Comment (felixgb) [reply schema/omega-services.async.api.yaml]: done!

**Review by rwalpole — APPROVED** [review]

**Pull request merged**